lib/test/unit_tests/test_adf_base.py

```diff
@@ -44,9 +45,12 @@ def tearDown(self):
         Remove log files (if they exist).
         """
 
-        #Remove log file if it exists:
-        if os.path.exists("ADF_debug.log"):
-            os.remove("ADF_debug.log")
+        debug_list = glob.glob("ADF_debug*")
```

Comment:
I might includ the word `log` in the glob string here:

```suggestion
        debug_list = glob.glob("ADF_debug*.log")
```